### PR TITLE
Add user_id to Google Analytics GTag

### PR DESCRIPTION
**Pull request by sean-wallace**
This commit adds user_id support for the gtag google analytics implementation.

### CODE REVIEW

File: analytical/templatetags/google_analytics_gtag.py [hunk 1]
```diff
@@ -51,8 +55,18 @@ def __init__(self):
...
+        identity = get_identity(context)
+        if identity is not None:
+            other_fields['user_id'] = identity
+
+        extra = '\n'.join([
+            GTAG_SET_CODE.format(key=key, value=value) for key, value in other_fields.items()
+        ])
         html = SETUP_CODE.format(
             property_id=self.property_id,
+            extra=extra
```
Comment (bittner) [inline analytical/templatetags/google_analytics_gtag.py:69]: A trailing comma might be nice on this line.

**Comment by sean-wallace** [comment]
> I believe, you need to extend the module documentation, don't you?

I've added a little something.

> What is the `l` -> `tl` name changes all about? If those are meant to be "meaningful names" can you use _actual_ self-explanatory names, please?

These were the simplest changes that would let `flake8` pass -- it looks like some new rules have come in since the last time it ran on master. I undid it since I don't know what the single letter variables stand for.

File: analytical/templatetags/google_analytics_gtag.py (hunk 1 above)
Comment (sean-wallace) [reply analytical/templatetags/google_analytics_gtag.py]: Done, thanks.

**Comment by bittner** [comment]
* `p` is the "position", I'd suggest to use `pos`.
* `l` is the "location", I'd suggest to use `loc`.

**Comment by sean-wallace** [comment]
All done, thanks for the help!

**Comment by bittner** [comment]
Thanks! :+1:

**Pull request merged**